tests/components/here_weather/const.py

```diff
@@ -0,0 +1,260 @@
+"""Constants for here_weather tests."""
+import herepy
+
+daily_simple_forecasts_json = {
```

Comment:
Please move the json to tests/fixtures directory as json files. Create a here_weather subdirectory.

Use the load_fixture test helper:
https://github.com/home-assistant/core/blob/66a5f24d4a4f5b39aa4dc42c25209c2dc71303c2/tests/common.py#L328-L332